mem-27-extra.xml

```diff
@@ -874,7 +874,7 @@ The speech concludes with the words (in Federation Standard): "We come in peace.
       <column name="antonyms"></column>
       <column name="see_also">{tlhIngan maH!:sen}</column>
       <column name="notes">This literally means "We are Klingon, may it continue". The English subtitles render this as "Remain Klingon!"</column>
-      <column name="notes_de">Dies bedeutet wörtlich "Wir sind Klingonen, möge es weitergehen". Die englischen Untertitel geben dies als "Remain Klingon!" [AUTOTRANSLATED]</column>
+      <column name="notes_de">Dies bedeutet wörtlich "Wir sind Klingonen, möge es weitergehen". Die englischen Untertitel geben dies als "Klingonen bleiben!"</column>
```

Comment:
The entry name is supposed to match what's in the subtitle, and the note here is explaining why this is different from the literal translation. As I understand it, your edit makes a contradiction which can be fixed in one of several ways:
1. Revert "Klingonen bleiben" back to "Remain Klingon!"
2. Change "englischen" to "deutschen", in which case either of "Wir bleiben Klingonen!" or "Klingonen bleiben!" should be changed to match whatever was actually used on Netflix. (The alternate version can be added as a German search tag if both are acceptable translations.)
3. Instead of "English subtitles", clarify by saying "a literal translation of the English subtitles...".
4. Note both what the German subtitles say (as in 2) *and* what the English subtitles say (as in 3), e.g., [the German subtitles say "Klingonen bleiben [or Wir Klingonen bleiben]!", a translation of the English subtitles which say "Remain Klingon!"], with the "Remain Klingon!" quote in English.

Since the show actually did have German subtitles, probably 2 makes the most sense (there's no need to drag English subtitles into it, if the German viewer has likely never seen them), but I leave it up to you.

Reply:
okay, just fixed it.

Reply:
Did you send another pull request? I don't see the fix.

Reply:
I'm not sure. I think I did. only for that single change.

Reply:
Which way did you fix it?